apps/academy-wep/components/CommunityStats.tsx

```
@@ -0,0 +1,50 @@
+		<div className="hidden md:block relative">
+			<div className="absolute inset-0 bg-gradient-to-br from-[#7CC635]/20 to-blue-500/20 rounded-full blur-xl transform rotate-6 scale-95"></div>
+			<div className="relative bg-white p-6 rounded-full shadow-lg border border-gray-100 w-48 h-48 flex items-center justify-center">
+				<div className="relative z-10 flex flex-col items-center">
+					<svg
+						width="64"
+						height="64"
+						viewBox="0 0 32 32"
+						fill="none"
+						xmlns="http://www.w3.org/2000/svg"
+						className="mb-3"
+					>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)